rootfs/conf/kops/helmfile.yaml

```diff
@@ -675,6 +675,8 @@ releases:
     default: "true"
   chart: "coreos-stable/kube-prometheus"
   version: "0.0.66"
+  values:
+    - "custom_values/kube-prometheus.grafana.dashboards.yaml"
```

Comment:
I like this solution, but I don't like the introduction of a new `custom_values` namespace. Can we just stick with `values/`?